climada/entity/impact_funcs/wildfire.py

```diff
@@ -0,0 +1,115 @@
+"""
+This file is part of CLIMADA.
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU Lesser General Public License as published by the Free
+Software Foundation, version 3.
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+---
+Define impact functions for WildFires.
+"""
+
+__all__ = ['IFWildfire']
+
+import logging
+import numpy as np
+
+from climada.entity.impact_funcs.base import ImpactFunc
+
+LOGGER = logging.getLogger(__name__)
+
+class IFWildfire(ImpactFunc):
+    """Impact function for wildfire."""
+
+    def __init__(self):
+        ImpactFunc.__init__(self)
+        self.haz_type = 'WFsingle'
+
+    def set_default_FIRMS(self, i_half=535.8, if_id=1):
+
+        """Using the formula of Emanuele 2011 but adjusted to wildfire.
+        As 295K resembles the lowest number of a FIRMS data set, i_thresh can
```

Comment:
what is `i_tresh`? It is not a kwargs of the function not is it explained what this is...

Reply:
I've adjusted the description